Cannot install kdeploy because of permission denied.

**Describe the bug**
When I tried to initialize, `npm install` output this error:
```
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/kaiostech/kdeploy.git
npm ERR! 
npm ERR! Warning: Permanently added the RSA host key for IP address '13.229.188.59' to the list of known hosts.
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: 无法读取远程仓库。(Unable to read remote repository.)
npm ERR! 
npm ERR! 请确认您有正确的访问权限并且仓库存在。(Please make sure you have the correct access rights and the repository exists.)
npm ERR! 
npm ERR! exited with error code: 128
```
Can I need to remove `kdeploy` from `package.json`?

**To Reproduce**
Steps to reproduce the behavior:
1. `git clone (URL)`
2. `cd KaiUI`
3. `npm install`
4. See error

**Browser/Emulator (please complete the following information):**
 - OS: Windows 10/Manjaro Linux
 - Browser: Pale Moon/Kaiosrt
 - Version: 28.6.1/Sep19

--- a/package.json
+++ b/package.json
@@ -14,12 +14,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "app:install": "kdeploy build install",
-    "app:uninstall": "kdeploy build uninstall",
-    "app:update": "npm run build && kdeploy build update",
-    "app:start": "kdeploy build start",
-    "app:stop": "kdeploy build stop"
+    "eject": "react-scripts eject"
   },
   "eslintConfig": {
     "extends": "react-app"
@@ -35,8 +30,5 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  },
-  "devDependencies": {
-    "kdeploy": "kaiostech/kdeploy"
   }
 }

